pkg/syncutil/aggregator/aggregator.go

```diff
@@ -98,6 +98,25 @@ func (b *GVKAgreggator) Upsert(k Key, gvks []schema.GroupVersionKind) error {
 	return nil
 }
 
+// List returnes the gvk set for a given Key.
+func (b *GVKAgreggator) List(k Key) map[schema.GroupVersionKind]struct{} {
```

Comment:
This is used for returning the GVKs for a given source key. I see I started using it in some of the testing for the cache manager to query the state of the aggregator.

absent this, it would be hard for us to do so without relying on the internals of the aggregator.